src/signals/incident-management/containers/Signaling/Signaling.tsx

```diff
@@ -111,15 +111,18 @@ const Signaling: FunctionComponent = () => {
       {heading}
       <Row>
         <StyledColumn span={6} wrap>
-          <GraphDescription
-            title={`Openstaande meldingen tot en met 2020`}
-            description={`
+          {totalOpen !== null ? (
```

Comment:
if its data or null, might as well go `totalOpen && (....`   vs   `totalOpen !== null ? (..) : null`

Reply:
That's not possible: if `totalOpen === 0` the component would not be rendered.

Reply:
in the `useMemo` above you use `total_signal_count || null`, if that number is `0` it will default to `null` as well, maybe that needs a more precise check in that case

Reply:
Sharp! Fixed it